sdcm/cluster.py

```diff
@@ -1734,7 +1734,7 @@ def install_package(self,
             pkg_cmd = 'zypper'
             package_name = f"{package_name}-{package_version}" if package_version else package_name
         else:
-            pkg_cmd = 'apt-get'
+            pkg_cmd = 'DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"'
```

Comment:
I would need more context why we are doing this here, cause we need to reinstall things by force ?

Reply:
when installing previous/later versions of scylla, scylla.yaml changes - this makes it to stay not touched. 
Need to do it automatically, installation hangs.

Reply:
the noninteractive part, I'm o.k., the rest of the options, I'm not sure it would fit all of the installation we are doing with function....

I would extend it, and pass the options only from that one or two places it's needed.

so we won't need to test all of the other usages with this command.